src/jvmTest/kotlin/wizard/GeneratedComposeAppProjectTest.kt

```diff
@@ -163,6 +166,32 @@ class GeneratedComposeAppProjectTest {
             }
         )
         val dir = projectInfo.writeToDir(workingDir)
+        val devicesJson = checkCommand(
+            dir = dir,
+            command = listOf(
+                "xcrun",
+                "simctl",
+                "list",
+                "devices",
+                "available",
+                "-j"
+            )
+        )
+        val devicesList = Json { ignoreUnknownKeys = true }.decodeFromString<XcodeDeviceResponse>(devicesJson)
+            .devices
+            .mapValues { entry -> entry.value
+                .filter { it.isAvailable }
+                .filter {
+                    if (true.toString().equals(other = System.getenv("CI"), ignoreCase = true)) {
+                        listOf("iphone 15", "iphone 14").any { device -> it.name.contains(device, true) }
+                    } else {
+                        true
+                    }
+                }
+            }
+            .values.flatten()
```

Comment:
BTW, are you sure we need a static types here?
We can use regular java JSON class to work with the devicesJson as with plain string-to string map

Reply:
Implemented using the `JSONObject`